src/main/java/org/folio/rest/impl/UserImportAPI.java

```diff
@@ -45,13 +45,13 @@
 import io.vertx.core.Promise;
 import io.vertx.core.Vertx;
 import io.vertx.core.http.HttpMethod;
-import io.vertx.core.impl.CompositeFutureImpl;
+import io.vertx.core.impl.future.CompositeFutureImpl;
```

Comment:
Please do not use internal class CompositeFutureImpl, this is not part of the Vert.x API: https://vertx.io/docs/apidocs/
Use https://github.com/folio-org/okapi/blob/master/okapi-common/src/main/java/org/folio/okapi/common/GenericCompositeFuture.java instead.

Reply:
Agreed. I'll make a PR to fix this.